source/github-helpers/index.ts

```diff
@@ -140,3 +140,8 @@ export async function isPermalink(): Promise<boolean> {
 export function isNotRefinedGitHubRepo(): boolean {
 	return !location.pathname.startsWith('/sindresorhus/refined-github/');
 }
+
+// https://github.com/github/markup/blob/cd01f9ec87c86ce5a7c70188a74ef40fc4669c5b/lib/github/markup/markdown.rb#L34
+export function isMarkdownFile(): boolean {
+	return /\.(mdx?|mkdn?|mdwn|mdown|markdown|litcoffee)$/.test(location.pathname);
```

Comment:
lit coffee is not markdown

Reply:
Quoting [the official website](https://coffeescript.org/#literate):

> If you name your file with a .litcoffee extension, you can write it as a Markdown document

And GitHub's code treats them as Markdown so I see no reason to change.

Reply:
Indeed the description for the format is _litera_lly "Literate CS is just Markdown"